### PR TITLE
Create quiz-git.md

### PR DESCRIPTION
**Pull request by ThomasBONNICART**
quiz git

### CODE REVIEW

File: quizzes/quiz-git.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+[x] git rebase
+[] git reset
+[] git commit --amend
+[] git reflog
+
+**Question 2:** How can you display the changes introduced by a single commit using Git?
+[] git show <commit_id>
+[] git diff <commit_id>
+[x] git log --patch <commit_id>
+[] git log -p <commit_id>
```
Comment (barraponto) [inline quizzes/quiz-git.md:13]:
```suggestion
**Question 2:** How can you display the changes introduced by a single commit using Git?
[x] git show <commit_id>
[] git diff <commit_id>
[x] git log --patch <commit_id>
[x] git log -p <commit_id>
```
i mean, all of these work. `git log -p` and `git log --patch` are literally the same command!

File: quizzes/quiz-git.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+[] Merging multiple branches simultaneously
+[] Navigating between previous commits
+[x] Finding the commit that introduced a bug
+
+
+
+
+
+
+
```
Comment (barraponto) [inline quizzes/quiz-git.md:32]:
```suggestion
```
just removing empty space

**Review by barraponto — CHANGES_REQUESTED** [review]

**Comment by ThomasBONNICART** [comment]
it's OK, I've made the changes you asked me to make

**Comment by barraponto** [comment]
great!

**Pull request merged**